earthfile2llb/converter.go

```diff
@@ -643,6 +643,29 @@ func (c *Converter) internalRun(ctx context.Context, args []string, secretKeyVal
 			return fmt.Errorf("Secret definition %s not supported. Must start with +secrets/ or be an empty string", secretKeyValue)
 		}
 	}
+	// Secret Files.
+	for _, secretFileDef := range secretFiles {
+		secretPath, secretID, ok := variables.ParseEscapedKeyValue(secretFileDef)
+		if !ok {
+			return fmt.Errorf("Invalid secret-file definition %s", secretFileDef)
```

Comment:
Nit: We should use lower-case for errors. Mind changing the other `fmt.Errorf`'s in this function too?

Reply:
tackled separately in https://github.com/earthly/earthly/pull/663